cmd/process-agent/app/check.go

```diff
@@ -137,7 +142,9 @@ func runCheck(cfg *config.AgentConfig, ch checks.Check) error {
 
 	time.Sleep(1 * time.Second)
 
-	printResultsBanner(ch.Name())
+	if !checkOutputJSON {
+		printResultsBanner(ch.Name())
+	}
```

Comment:
Sanity check - on line 151 below - when we return an error - this does end up in stderr? 🙇

Reply:
Yep! We're only suppressing the console output for seelog here, so everything else will get printed.

I tested erroring unconditionally to ensure that cobra prints to `stderr` when returning an error, and it did, so we are good on that.